[BUG] Failed to save transcription to history: no such table: transcription_history

## Bug Description

The app does not save entries to the history table. And the following error appears in the console:

```
[2025-11-11T15:11:40Z ERROR handy_app_lib::actions] Failed to save transcription to history: no such table: transcription_history
```

The history.db file is empty:

```
0 -rw-r--r-- 1 anon anon 0 11 nov 19:09 /home/anon/.local/share/com.pais.handy/history.db
```

## System Information

**App Version:** 0.6.0

**Operating System:**
**CPU:**
**GPU:**

Operating System: EndeavourOS 
KDE Plasma Version: 6.4.5
KDE Frameworks Version: 6.19.0
Kernel Version: 6.17.3-arch2-1 (64-bit)
Graphics Platform: Wayland
Processors: 20 × 13th Gen Intel® Core™ i7-13700H
Memory: 32 GiB of RAM (31,0 GiB usable)
Graphics Processor 1: Mesa Intel® Iris® Xe Graphics
Graphics Processor 2: NVIDIA GeForce RTX 4050 Laptop GPU/PCIe/SSE2

---

Love the app by the way. I'm using it on Wayland and it mostly works. Just other issues that have already been brought up; in some apps it only work when the overlay is disabled (seems only Chrome/Electron-based apps work fine with the overlay) and in some apps it eats the first character. But not always. I've also had it hang a few times. And I have to use `WEBKIT_DISABLE_DMABUF_RENDERER=1` to run it.

Yes I see this error: `[2025-12-03][08:19:14][handy_app_lib::actions][ERROR] Failed to save transcription to history: no such table: transcription_history`. Happy to test a branch that implements a fix on my machine.

--- a/README.md
+++ b/README.md
@@ -83,6 +83,14 @@ This project is actively being developed and has some [known issues](https://git
 
 - Limited or no support for Wayland display server
 
+**Overlay & Pasting Issues (Linux):**
+
+- The recording overlay window can interfere with pasting transcribed text into target applications on Linux (X11)
+- **Solution:** Open **Settings > Advanced** and set **"Overlay Position"** to **"None"** to disable the overlay
+- Enable **"Audio Feedback"** (also in Advanced) if you still want audible confirmation of recording state
+- Users who upgrade from older versions or import settings from other platforms may need to manually apply this change
+
+
 ### Platform Support
 
 - **macOS (both Intel and Apple Silicon)**

